src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -54,8 +54,10 @@ public static void Verify(string expectedOperationTree, string actualOperationTr
         {
             char[] newLineChars = Environment.NewLine.ToCharArray();
             string actual = actualOperationTree.Trim(newLineChars);
+            actual = actual.Replace("\"", "\"\"");
```

Comment:
>actual [](start = 12, length = 6)

:memo: This change makes it easier to work with IOperations tests. You can now copy/paste the Actual without having to fiddle with double-quotes.